### DISCUSSION TITLE
Help setting up Netbox with a reverse proxy

### DISCUSSION THREAD

**Discussion by kobyon** [netbox-community/netbox#15922] [Help Wanted!] [answered]
Hello,
I am trying to use my existing webserver to serve the netbox webpage.
I have installed the netbox server on one server, on this server I don't want to have a separate instance of nginx or apache.
On another server I am running Nginx Proxy Manager. 

My current nginx proxy manager config looks like this:
```
client_max_body_size 25m;

location /static/ {
    proxy_pass http://ip-for-netbox-server:8001/netbox/netbox/static/;
}

location / {
    proxy_pass http://ip-for-netbox-server:8001;
    proxy_set_header X-Forwarded-Host $http_host;
    proxy_set_header X-Real-IP $remote_addr;
    proxy_set_header X-Forwarded-Proto $scheme;
}
```
This configuration works to server the web page, except when I connect I get this error message
![Screenshot 2024-05-02 153329](https://github.com/netbox-community/netbox/assets/77970971/2c304a71-ecc9-476a-9ed5-99dbf8d9bd9f)

I believe the error lies in the "location /static/".  Can anyone help me create a working nginx config?

**Comment by a084ed22** [comment, marked as answer]
the files under /static/ are not served via gunicorn, but directly by the reverse proxy when this is installed on the same server as netbox. Since you have a different server, apache/nginx won't be able to access them directly. I'm uncertain whether it's possible to force gunicorn to serve the static assets.
